### ISSUE TITLE
How about ahk v2 version?

### ISSUE THREAD

**Issue by krasnovpro** [FuPeiJiang/VD.ahk#19]

**Comment by FuPeiJiang** [comment]
https://github.com/FuPeiJiang/VD.ahk/tree/v2_port
used 
https://github.com/FuPeiJiang/ahk_to_v2.ah2: pass [_VD.ahk](https://github.com/FuPeiJiang/ahk_to_v2.ah2/blob/main/_VD.ahk) through [bruh.ah2](https://github.com/FuPeiJiang/ahk_to_v2.ah2/blob/main/bruh.ah2) (it reads from `A_Clipboard` as input)

**Comment by rikrdo89** [comment]
when using AHK v2.0 and the V2_port script, I get the following error: "no method named this._dll_CreateDesktop"

![image](https://user-images.githubusercontent.com/24930604/209855057-b7021278-d7e1-4374-bac3-848127ae6311.png)

**Comment by FuPeiJiang** [comment]
thank you for this bug report: this doesn't only happen in v2_port, this also happens in v1, it's a somehow missed line of code for windows 11

https://github.com/FuPeiJiang/VD.ahk/commit/5d4bed4359c6f15e02939f4875174a91e6a60035
hope this fixes it

**Comment by micjoh** [comment]
Is this going to be the new main version?

**Comment by FuPeiJiang** [comment]
> Is v2 going to be the new main version?

not really, because I will be updating both v1 and v2 at the same time, updates are rare, so that's not a problem ?

**Comment by micjoh** [comment]
No problem at all - I just wondered
This is a marvelous little program that is of great utility.
One I can't be without so very grateful for what you are doing :-)
